src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/JsonValueConverterUri.cs

```diff
@@ -6,10 +6,10 @@ namespace System.Text.Json.Serialization.Converters
 {
     internal sealed class JsonConverterUri : JsonConverter<Uri>
     {
-        public override Uri Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        public override Uri Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions? options)
```

Comment:
Same issue here/elsewhere (`options` shouldn't be nullable). I am going to stop mentioning this.